assets/js/script.js

```diff
@@ -1,130 +1,149 @@
-(function ($) {
-  'use strict';
-
-  // Background-images
-  $('[data-background]').each(function () {
-    $(this).css({
-      'background-image': 'url(' + $(this).data('background') + ')'
+(function($) {
+    'use strict';
+
+    // Background-images
+    $('[data-background]').each(function() {
+        $(this).css({
+            'background-image': 'url(' + $(this).data('background') + ')'
+        });
     });
-  });
 
 
-  // Accordions
-  $('.collapse').on('shown.bs.collapse', function () {
-    $(this).parent().find('.ti-plus').removeClass('ti-plus').addClass('ti-minus');
-  }).on('hidden.bs.collapse', function () {
-    $(this).parent().find('.ti-minus').removeClass('ti-minus').addClass('ti-plus');
-  });
+    // Accordions
+    $('.collapse').on('shown.bs.collapse', function() {
+        $(this).parent().find('.ti-plus').removeClass('ti-plus').addClass('ti-minus');
+    }).on('hidden.bs.collapse', function() {
+        $(this).parent().find('.ti-minus').removeClass('ti-minus').addClass('ti-plus');
+    });
```

Comment:
I'm curious why the spacing was changed here. Is it for a particular reason? Normally it would be no big deal, but small changes like this can cause tricky merge conflicts when rebasing from an upstream.